src/coreComponents/mesh/FaceManager.cpp

```diff
@@ -235,6 +236,11 @@ void FaceManager::computeGeometry( NodeManager const & nodeManager )
                                                                          m_faceNormal[ faceIndex ] );
 
   } );
+
+  elemManager.forElementSubRegions< CellElementSubRegion, FaceElementSubRegion >( [&] ( auto const & subRegion )
+  {
+    subRegion.calculateElementCenters( X );
+  } );
```

Comment:
I find a bit weird that a function in `FaceManager` called `computeGeometry` takes care of computed element centers of the subregions. Also, do the fracture elements even exist at this point? I am assuming this occurs way before the event loop in which the surfacegenerator creates the fractures.